layouts/shortcodes/feature-gate-list.html

```diff
@@ -48,13 +48,14 @@
 
     {{- if $shouldDisplayThisFeatureGate -}}
       {{- with $featureGateFile.Content -}}
-      <li>
+      <dt id="{{ $featureGateName }}"><code class="feature-gate-name">{{ $featureGateName }}</code></dt>
```

Comment:
I might:
```suggestion
      <dt id="feature-{{ $featureGateName }}"><code class="feature-gate-name">{{ $featureGateName }}</code></dt>
```

to avoid the risk of a collision with some other site feature. However, don't let this block a merge.

Reply:
I initially considered using a prefix as well, but then dismissed the idea. I guess anchors can be used to link to feature gate descriptions instead of linking to the general page, like here.

https://github.com/kubernetes/website/blob/e9518c5f28ea8d460b7aa1f495015be22d16f4cd/content/en/docs/concepts/architecture/leases.md?plain=1#L92-L95

The code will turn to ⬇️ 

```diff

- You can disable API server identity leases by disabling the `APIServerIdentity`
- [feature gate](/docs/reference/command-line-tools-reference/feature-gates/).
+ You can disable API server identity leases by disabling the [`APIServerIdentity`
+ feature gate](/docs/reference/command-line-tools-reference/feature-gates/#APIServerIdentity).

```